### ISSUE TITLE
Mopidy randomly crashes

### ISSUE THREAD

**Issue by dirkgroenen** [mopidy/mopidy#1383]
The last few days/weeks I experienced some strange behavior with Mopidy. I'm running a setup with Mopidy en Mopify on a Raspberry running Raspbian. For some reason it sometimes randomly crashes when a song has ended. 

The output of `service mopidy status` shows me the following:

```
Jan 02 21:59:10 raspberrypi mopidy[2359]: File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 201, in _actor_loop
Jan 02 21:59:10 raspberrypi mopidy[2359]: response = self._handle_receive(message)
Jan 02 21:59:10 raspberrypi mopidy[2359]: File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 295, in _handle_receive
Jan 02 21:59:10 raspberrypi mopidy[2359]: return callee(*message['args'], **message['kwargs'])
Jan 02 21:59:10 raspberrypi mopidy[2359]: File "/usr/lib/python2.7/dist-packages/mopidy/core/listener.py", line 34, in on_event
Jan 02 21:59:10 raspberrypi mopidy[2359]: getattr(self, event)(**kwargs)
Jan 02 21:59:10 raspberrypi mopidy[2359]: TypeError: playback_state_changed() got an unexpected keyword argument 'old_state'
Jan 02 21:59:10 raspberrypi mopidy[2359]: Fatal Python error: GC object already tracked
Jan 02 21:59:10 raspberrypi systemd[1]: mopidy.service: main process exited, code=killed, status=6/ABRT
Jan 02 21:59:10 raspberrypi systemd[1]: Unit mopidy.service entered failed state.
```

Anyone who has any idea of what's happening?

**Comment by adamcik** [comment]
Some bad frontend/backend for the `getattr` line, though not clear which one. So look for something with the wrong argument names relative to the events. aa010e03e995d68913a071793f1847a1ac8cd356 should fix this in develop, and should probably be backported to 1.1.x

The following "GC object already tracked" at shutdown on the otherhand looks like something in some bindings, not mopidy itself.

**Comment by jodal** [comment]
Fix cherry-picked to release-1.1 branch.
